gatsby-config.js

```diff
@@ -13,167 +13,525 @@
 module.exports = {
   siteMetadata: {
     home: {
-      title: 'Adobe Document Services',
-      path: '/document-services/'
+      title: 'Products',
```

Comment:
@adityaganotra fyi the default home if none is provided is "Products" linking to /apis so you don't need to  set `home` in this case.